src/components/CalendarMonthOption.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { PropTypes } from 'react';
+import shallowCompare from 'react-addons-shallow-compare';
+import momentPropTypes from 'react-moment-proptypes';
+import moment from 'moment';
+
+export const TOUCHSTART_TIMEOUT = 200;
+
+const propTypes = {
+  month: momentPropTypes.momentObj,
+  modifiers: PropTypes.arrayOf(PropTypes.string),
+  onMonthClick: PropTypes.func,
+  onMonthMouseDown: PropTypes.func,
+  onMonthMouseUp: PropTypes.func,
+  onMonthMouseEnter: PropTypes.func,
+  onMonthMouseLeave: PropTypes.func,
+  onMonthTouchStart: PropTypes.func,
+  onMonthTouchEnd: PropTypes.func,
+  onMonthTouchTap: PropTypes.func,
+};
+
+const defaultProps = {
+  month: moment(),
```

Comment:
You're right, I'm changing a little the behaviour. This is an internal component that renders the cell with the selectable month. Here the month will always be needed as parameter so I think maybe it's better that it will always be required, instead of having a default value.